lib/Controller/Text2ImageController.php

```diff
@@ -43,4 +54,23 @@ public function showGenerationPage(?string $imageGenId, ?bool $forceEditMode = f
 
 		return new TemplateResponse(Application::APP_ID, 'imageGenerationPage');
 	}
+
+	/**
+	 * @return DataResponse
+	 */
+	#[NoAdminRequired]
+	#[NoCSRFRequired]
+	public function getPromptHistory(): DataResponse {
```

Comment:
History is not part of the API?

Reply:
Not the prompt history, it is only used by the smart pickers.
This is not related with task history.

Reply:
ah